fe2-android/app/src/main/java/com/github/dedis/popstellar/utility/handler/data/ReactionHandler.java

```diff
@@ -0,0 +1,70 @@
+package com.github.dedis.popstellar.utility.handler.data;
+
+import com.github.dedis.popstellar.model.network.method.message.data.socialmedia.AddReaction;
+import com.github.dedis.popstellar.model.network.method.message.data.socialmedia.DeleteReaction;
+import com.github.dedis.popstellar.model.objects.Channel;
+import com.github.dedis.popstellar.model.objects.Reaction;
+import com.github.dedis.popstellar.model.objects.security.MessageID;
+import com.github.dedis.popstellar.model.objects.security.PublicKey;
+import com.github.dedis.popstellar.model.objects.view.LaoView;
+import com.github.dedis.popstellar.repository.LAORepository;
+import com.github.dedis.popstellar.repository.SocialMediaRepository;
+import com.github.dedis.popstellar.utility.error.InvalidMessageIdException;
+import com.github.dedis.popstellar.utility.error.UnknownLaoException;
+
+import javax.inject.Inject;
+
+import timber.log.Timber;
+
+/** Reaction to chirps handler class */
+public class ReactionHandler {
+
+  public static final String TAG = ReactionHandler.class.getSimpleName();
+
+  private final LAORepository laoRepo;
+  private final SocialMediaRepository socialMediaRepo;
+
+  @Inject
+  public ReactionHandler(LAORepository laoRepo, SocialMediaRepository socialMediaRepo) {
+    this.laoRepo = laoRepo;
+    this.socialMediaRepo = socialMediaRepo;
+  }
+
+  public void handleAddReaction(HandlerContext context, AddReaction addReaction)
```

Comment:
I just noticed that documenation is missing here, also for `handleDeleteReaction`